tasks/install-server.yml

```diff
@@ -1,8 +1,17 @@
 ---
+- name: Compose the Minio server download base url
+  set_fact:
+    _minio_server_download_base_url: "https://dl.minio.io/server/minio/release/linux-{{ go_arch_map[ansible_architecture] | default(ansible_architecture) }}"
```

Comment:
Rather than use this `go_arch_map` evaluation here, use it in a vars entry.

Similar to we do it in [ansible-prometheus](https://github.com/cloudalchemy/ansible-prometheus/blob/master/vars/main.yml).

Reply:
Actually, let me send that as a separate refactoring PR.

Reply:
Effectively, I will wait for your PR to be merged.

Reply:
Yea, first need to fix CI with https://github.com/atosatto/ansible-minio/pull/33. :smile: